### ISSUE TITLE
Staging Review finding: Table semantics may be announced for "Important Warning"

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#98162]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

This ticket was part of a merged Staging Review with MHV Medications [List Filter](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88958) and [Prescription Information](https://github.com/department-of-veterans-affairs/va.gov-team/issues/88952)

## Product Information
**Team:** MHV Medications
**Product:** Medications
**Feature:** List Filter & Prescription Information

## Findings details
**VA.gov Experience Standard - issue:** [User encounters unnecessary repetition or redundancy.](https://design.va.gov/about/experience-standards/#user-encounters-no-repetition-or-redundancy)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On a prescription details page (eg. [Salmeterol](https://staging.va.gov/my-health/medications/prescription/23301674/documentation?ndc=00093721410) for test user KARSYN.GPSYSASYNCTWO@ID.me), there's a section of content with a heading of "Important warning." That content is wrapped in a `<table>` and contains one table cell, which contains all of the content of the warning.

Some screen readers will be smart enough to ignore the table semantics here, since there's only one cell. Other screen readers may still announce the table structure as though it's a data table, which may lead to a lot of extra verbosity before a user ever gets to the warning.

<details><summary>Link, screenshot or steps to recreate</summary>

![Box with heading "Important Warning" followed by descriptions of possible rug side effects and interactions.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/88952-Brian-1.png?raw=true)

</details>

## Recommended action

If it's possible to strip out the `table`, `tr`, `td`, etc entirely and display the warning as a `div` instead, that would be ideal. If that's not practical from the source data for these pages, then the next best thing would be to insert a `role="presentation"` attribute for the `table`. That will instruct screen readers to ignore the table semantics and just announce it as straight content.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by TannerBird98** [comment]
Work is in progress now and should be completed within the week.  [https://jira.devops.va.gov/browse/MHV-64831](url)
